### ISSUE TITLE
R package repository smoke test fails due to argon2 package

### ISSUE THREAD

**Issue by edwardchalstrey1** [alan-turing-institute/data-safe-haven#1430]
## :white_check_mark: Checklist

- [x] I have searched open and closed issues for duplicates.
- [x] This is a problem observed when **managing** a Data Safe Haven.
- [x] I can reproduce this with the [latest version](https://github.com/alan-turing-institute/data-safe-haven/releases).
- [x] I have read through the [documentation](https://alan-turing-institute.github.io/data-safe-haven/).
- [x] This isn't an open-ended question (open a [discussion](https://github.com/alan-turing-institute/data-safe-haven/discussions) if it is).

## :computer: System information

- **Data Safe Haven version:** 4.0.3
- **Operating system details:** macOS

## :cactus: Powershell module versions

```
2023-03-21 15:11:09 [SUCCESS]: [✔] Powershell version: 7.2.6
2023-03-21 15:11:09 [SUCCESS]: [✔] Az.MonitoringSolutions module version: 0.1.0
2023-03-21 15:11:09 [SUCCESS]: [✔] Az.Accounts module version: 2.9.0
2023-03-21 15:11:09 [SUCCESS]: [✔] Az.Network module version: 4.18.0
2023-03-21 15:11:09 [SUCCESS]: [✔] Poshstache module version: 0.1.10
2023-03-21 15:11:10 [SUCCESS]: [✔] Az.Resources module version: 6.0.1
2023-03-21 15:11:10 [SUCCESS]: [✔] Microsoft.Graph.Authentication module version: 1.17.0
2023-03-21 15:11:10 [SUCCESS]: [✔] Az.DataProtection module version: 0.4.0
2023-03-21 15:11:10 [SUCCESS]: [✔] Az.Storage module version: 4.7.0
2023-03-21 15:11:10 [SUCCESS]: [✔] Az.KeyVault module version: 4.6.0
2023-03-21 15:11:10 [SUCCESS]: [✔] Az.OperationalInsights module version: 3.1.0
2023-03-21 15:11:10 [SUCCESS]: [✔] Az.Automation module version: 1.7.3
2023-03-21 15:11:10 [SUCCESS]: [✔] Az.Compute module version: 4.29.0
2023-03-21 15:11:10 [SUCCESS]: [✔] Microsoft.Graph.Identity.DirectoryManagement module version: 1.10.0
2023-03-21 15:11:10 [SUCCESS]: [✔] Az.RecoveryServices module version: 5.4.1
2023-03-21 15:11:10 [SUCCESS]: [✔] Powershell-Yaml module version: 0.4.2
2023-03-21 15:11:10 [SUCCESS]: [✔] Az.PrivateDns module version: 1.0.3
2023-03-21 15:11:10 [SUCCESS]: [✔] Microsoft.Graph.Applications module version: 1.17.0
2023-03-21 15:11:10 [SUCCESS]: [✔] Az.Monitor module version: 3.0.1
2023-03-21 15:11:10 [SUCCESS]: [✔] Az.Dns module version: 1.1.2
```

## :no_entry_sign: Describe the problem

Running `bats run_all_tests.bats` has the following error:

![Screenshot 2023-03-21 at 15 07 44](https://user-images.githubusercontent.com/5486164/226651183-ffc475ad-d47a-43a5-8cfe-3290234a0ab8.png)

**Comment by JimMadge** [comment]
Looks like the package no longer exists. https://cran.r-project.org/web/packages/argon2/index.html

**Comment by edwardchalstrey1** [comment]
In the test, switch argon2 for something else on the allowlist

**Comment by JimMadge** [comment]
closed in #1421
